src/AgOledDisplay.cpp

```diff
@@ -219,23 +307,31 @@ void OledDisplay::showDashboard(const char *status) {
   }
 
   char strBuf[10];
+  const int icon_pos_x = 64;
 
   DISP()->firstPage();
   do {
     DISP()->setFont(u8g2_font_t0_16_tf);
+    showTempHum(false);
+
     if ((status == NULL) || (strlen(status) == 0)) {
-      showTempHum(false);
+      int rssi = WiFi.RSSI();
+      int icon_type = WIFI_ICON_25;
+      if (rssi > -40) {
+        icon_type = WIFI_ICON_100;
+      } else if (rssi > -67) {
+        icon_type = WIFI_ICON_75;
+      } else if (rssi > -70) {
+        icon_type = WIFI_ICON_50;
+      } else {
+        icon_type = WIFI_ICON_25;
+      }
+      showIcon(icon_pos_x, 0, &wifi_icons[icon_type]);
     } else {
-      String strStatus = "Show status: " + String(status);
-      logInfo(strStatus);
-
-      int strWidth = DISP()->getStrWidth(status);
-      DISP()->drawStr((DISP()->getWidth() - strWidth) / 2, 10, status);
-
-      /** Show WiFi NA*/
       if (strcmp(status, "WiFi N/A") == 0) {
```

Comment:
Propose to introduce an enum and avoid implementation of business logic based on strings that are possibly localized in the future / shown to the user / subject to change.

Reply:
Ok, I will create global string instead. Because we are also meed to string message `Add to dashboard` along side with icons

Reply:
Still think that we need an enum

Reply:
OK